python/generate-short-url/README.md

```diff
@@ -0,0 +1,61 @@
+# 📱 Validate phone number and get it's country information
+
+A Python Cloud Function that generates short URLs with help of Bitly or Tinyurl.
+
+_Example input:_
+
+```json
+{
+	"provider": "tinyurl", 
+	"url": "https://github.com/open-runtimes/examples",
+	"api_key": "API_KEY"
+}
+```
+
+
+_Example output:_
+
+
+```json
+{
+    "response": {
+        "success": true,
+        "url": "https://tinyurl.com/xxxxxxx"
+    },
+    "stdout": ""
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
+
+- **APPWRITE_FUNCTION_ENDPOINT** - Endpoint of your Appwrite server
+- **APPWRITE_FUNCTION_API_KEY** - Appwrite API Key
+- **APPWRITE_FUNCTION_PROJECT_ID** - Appwrite project ID. If running on Appwrite, this variable is provided automatically.
```

Comment:
We dont have these variables here. We should have `TINYURL_API_KEY` and `BITLY_API_KEY` here.